pennylane_lightning/core/src/utils/TestHelpers.hpp

```diff
@@ -288,6 +288,19 @@ auto createZeroState(size_t num_qubits) -> TestVector<ComplexT> {
     return res;
 }
 
+/**
+ * @brief create |+>^N
+ */
+template <typename ComplexT>
+auto createPlusState_(size_t num_qubits) -> TestVector<ComplexT> {
```

Comment:
I added this modified version of `createPlusState()` as I needed a `ComplexT` based template for the Kokkos unittest.
Should I try to replace the `PrecisionT` version?